sdk/core/azure-core-amqp/src/impl/rust_amqp/rust_amqp/azure_core_amqp/src/fe2o3/connection.rs

```diff
@@ -158,7 +159,24 @@ impl AmqpConnectionApis for Fe2o3AmqpConnection {
                 info.map(|i| i.into()),
             ))
             .await
-            .map_err(AmqpConnection::from)?;
-        Ok(())
+            .map_err(AmqpConnection::from);
+        // If we're closing with an error, then we might get the transport error back before we get the error back.
+        // that's ok.
+        match res {
+            Ok(_) => Ok(()),
+            Err(e) => match e.0 {
+                fe2o3_amqp::connection::Error::TransportError(e) => {
+                    info!(
```

Comment:
IMO, feels more like a debug trace if it's ignored.

Reply:
That makes sense. Updating.